scripts/Dockerfile

```diff
@@ -1,5 +1,7 @@
 ARG LOGSTASH_VERSION
 FROM docker.elastic.co/logstash/logstash-oss:${LOGSTASH_VERSION}
+USER root
+RUN apt-get update && apt-get install -y python
```

Comment:
It looks like this is failing. Maybe you need to specify the path?

Somewhat relatedly, this script is install Python in order to output the OpenSearch version. Does any automation need this string in a certain format? If it is purely informational, it might be cleaner to avoid Python and just use something like `grep`.

```
echo $(curl -s $SERVICE_URL | grep number")
```

Or maybe just output the full data from the version request.

Reply:
Replacing the python command with grep. The exact version number is required in the integration test to work around feature incompatibilities between OpenSearch 1.x and 2.x